### PR TITLE
Added panel regarding ammount of HPA

### PR DESCRIPTION
**Pull request by yuvalweber**
added panel regarding HPA current/max replicas

Relates to #3911

### CODE REVIEW

**Review by zroubalik — CHANGES_REQUESTED** [review]
Hi, could you please also update the Dashboard to target metrics exposed in KEDA Operator, we have just recently changed this a little bit. KEDA Operator is the ultimate source of prom metrics (prom metrics from Metrics Server are deprecated and will be removed).

There are the same metrics exposed by Operator, but they are slightly renamed, see for details: https://github.com/kedacore/keda/issues/3919

**Comment by yuvalweber** [comment]
Added 😄 @zroubalik

**Comment by tomkerkhove** [comment]
Would you mind posting a screenshot please?

**Review by tomkerkhove — APPROVED** [review]

File: config/grafana/keda-dashboard.json [hunk 1]
```diff
@@ -703,21 +803,25 @@
...
         "current": {
           "selected": true,
-          "text": ["http-demo"],
-          "value": ["http-demo"]
+          "text": [
+            "nginx-scale"
+          ],
+          "value": [
+            "nginx-scale"
+          ]
```
Comment (tomkerkhove) [inline config/grafana/keda-dashboard.json:811]: Let's use `keda-demo`

File: config/grafana/keda-dashboard.json [hunk 2]
```diff
@@ -755,21 +859,21 @@
       {
         "current": {
           "selected": false,
-          "text": "s0-prometheus-container_cpu_usage_seconds_total",
-          "value": "s0-prometheus-container_cpu_usage_seconds_total"
+          "text": "s0-prometheus-nginx_connections_active_keda",
+          "value": "s0-prometheus-nginx_connections_active_keda"
```
Comment (tomkerkhove) [inline config/grafana/keda-dashboard.json:863]: Let's remove reference to nginx

**Comment by yuvalweber** [comment]
> Would you mind posting a screenshot please?
Screen Shot 2022-11-30 at 10 18 52

**Comment by tomkerkhove** [comment]
Do we want to see just over time or also current? We might want to add a gauge?
https://grafana.com/docs/grafana/v9.0/visualizations/#gauge

**Comment by yuvalweber** [comment]
> Do we want to see just over time or also current? We might want to add a gauge? https://grafana.com/docs/grafana/v9.0/visualizations/#gauge

Do you want to add a new one or to replace the current one with gauge?

**Comment by tomkerkhove** [comment]
It's just an idea but I would add both

**Comment by yuvalweber** [comment]
hey @zroubalik can you comment regarding this change?

**Comment by yuvalweber** [comment]
Screen Shot 2022-12-02 at 9 12 00
 also tried to added this as a gauge and I want your opinions about the look (I liked the time based better because I found the option using the graph to show the current vs max when it let you shows what the max value is).
But please also comment on this and if you think it's helpful I'll upload it so we could deliver the grafana dashboard to users 😄  @tomkerkhove

**Comment by tomkerkhove** [comment]
I think both are perfect - Left shows current status easily, right you can see it over time if you want more details.

What are your thoughts @zroubalik @JorTurFer ?

**Review by zroubalik — APPROVED** [review]
Looking good.

**Comment by yuvalweber** [comment]
So I'll add the second panel today to the PR and merge ok?
@tomkerkhove @zroubalik

**Comment by zroubalik** [comment]
> So I'll add the second panel today to the PR and merge ok? @tomkerkhove @zroubalik

yeah :)

**Comment by yuvalweber** [comment]
I can't seem to be authorize to merge to this branch so whenever you can please merge 😄

**Comment by tomkerkhove** [comment]
> I can't seem to be authorize to merge to this branch so whenever you can please merge 😄

That is by design :)

Thank you!

**Comment by tomkerkhove** [comment]
@yuvalweber Do you have an example screenshot of the full dashboard that we can add to our release blog post?

**Comment by yuvalweber** [comment]
yeah of course I'll do it today @tomkerkhove

**Comment by tomkerkhove** [comment]
@yuvalweber Did you manage to create one? If not, no problem.

**Comment by yuvalweber** [comment]
Sorry for late response.
This is what I have @tomkerkhove 
image

**Pull request merged**